### PR TITLE
*: add table storage info status api

### PR DESCRIPTION
**Pull request by jianzhiyao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

issue: https://github.com/pingcap/tidb/issues/4989
pull request: https://github.com/pingcap/tidb/pull/22859

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-monitoring-api.md [hunk 1]
```diff
@@ -93,3 +94,48 @@ Metrics 接口用于监控整个集群的状态和性能。
 - 如果使用其他方式部署 TiDB 集群，在使用 metrics 接口前，需先[部署 Prometheus 和 Grafana](/deploy-monitoring-services.md)。
 
 成功部署 Prometheus 和 Grafana 之后，[配置 Grafana](/deploy-monitoring-services.md)。
+
+## 存储信息接口
```
Comment (xhebox) [inline tidb-monitoring-api.md:98]: Maybe as a subsection of `### TiDB Server`?

**Review by xhebox — APPROVED** [review]

File: tidb-monitoring-api.md [hunk 2]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
+#### 存储信息
+
+以下示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，并判断该 TiDB Server 是否存活。结果以 **JSON** 格式返回：
```
Comment (TomShawn) [inline tidb-monitoring-api.md:43]:
```suggestion
以下示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，并判断该 TiDB Server 是否存活。结果以 JSON 格式返回：
```

File: tidb-monitoring-api.md [hunk 3]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
...
+
+{{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline tidb-monitoring-api.md:45]:
```suggestion
```

File: tidb-monitoring-api.md [hunk 4]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
...
+
+以下示例中，通过访问 `http://${host}:${port}/schema_storage/${db}/${table}` 获取指定数据表的存储信息。结果以 **JSON** 格式返回：
+
```
Comment (TomShawn) [inline tidb-monitoring-api.md:48]:
```suggestion

{{< copyable "shell-regular" >}}

```

File: tidb-monitoring-api.md [hunk 5]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
+#### 存储信息
+
+以下示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，并判断该 TiDB Server 是否存活。结果以 **JSON** 格式返回：
+
+{{< copyable "shell-regular" >}}
+
+以下示例中，通过访问 `http://${host}:${port}/schema_storage/${db}/${table}` 获取指定数据表的存储信息。结果以 **JSON** 格式返回：
```
Comment (TomShawn) [inline tidb-monitoring-api.md:47]:
```suggestion
以下示例中，通过访问 `http://${host}:${port}/schema_storage/${db}/${table}` 获取指定数据表的存储信息。结果以 JSON 格式返回：
```

File: tidb-monitoring-api.md [hunk 6]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
...
+    "table_rows": 0, 
+    "avg_row_length": 0, 
+    "data_length": 0, 
+    "max_data_length": 0, 
+    "index_length": 0, 
+    "data_free": 0
+}
+```
+
+```
```
Comment (TomShawn) [inline tidb-monitoring-api.md:66]:
```suggestion
```bash
```

File: tidb-monitoring-api.md [hunk 7]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
+#### 存储信息
+
+以下示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，并判断该 TiDB Server 是否存活。结果以 JSON 格式返回：
+
+以下示例中，通过访问 `http://${host}:${port}/schema_storage/${db}/${table}` 获取指定数据表的存储信息。结果以 JSON 格式返回：
```
Comment (TomShawn) [inline tidb-monitoring-api.md:45]:
```suggestion
```

File: tidb-monitoring-api.md [hunk 8]
```diff
@@ -35,6 +38,50 @@ curl http://127.0.0.1:10080/status
 }
 ```
 
+#### 存储信息
+
+以下示例中，通过访问 `http://${host}:${port}/status` 获取当前 TiDB Server 的状态，并判断该 TiDB Server 是否存活。结果以 JSON 格式返回：
```
Comment (TomShawn) [inline tidb-monitoring-api.md:43]: @jianzhiyao 这里看起来少了一个示例？

**Review by TomShawn — APPROVED** [review]

**Pull request merged**